proto/buf/alpha/registry/v1alpha1/doc.proto

```diff
@@ -377,6 +387,40 @@ message FieldOptions {
   int32 jstype = 4;
 }
 
+// Option provides information for a descriptor option that can be show in documentations.
+message Option {
```

Comment:
Given the package this is in (registry wide), the name `Option` might be a bit too vague. How about `DescriptorOption`?

Reply:
Good call, renamed to `DescriptorOption`